diagrams/generate.dart

```diff
@@ -277,10 +277,18 @@ class DiagramGenerator {
   /// The class that the app runs as.
   static const String appClass = 'io.flutter.api.diagrams';
 
+  /// The path to the top of the repo.
   static String get projectDir {
     return path.joinAll(path.split(path.absolute(path.fromUri(Platform.script)))..removeLast());
   }
 
+  /// The output asset directory for all the categories.
+  static String get assetDir {
+    return path.joinAll(path.split(projectDir)
+      ..removeLast()
+      ..add('assets'));
```

Comment:
there's some misleading indentation here. Looks like the cascade applies to the return value of `path.joinAll`, but it's actually applying to the argument.

Reply:
Fixed.